spec/models/queue_item_spec.rb

```diff
@@ -31,6 +50,34 @@
       end
     end
 
+    describe '#rating=' do
+      it 'changes the rating of the review when there is a review present' do
+        review = Fabricate(:review, author: user, video: video, rating: 3)
+        queue_item = Fabricate(:queue_item, user:user, video:video)
+
+        queue_item.rating = 2
+
+        expect(Review.first.rating).to eq(2)
+      end
+
+      it 'removes the rating of the review if there is a review present' do
+        review = Fabricate(:review, author: user, video: video, rating: 3)
+        queue_item = Fabricate(:queue_item, user:user, video:video)
```

Comment:
Agreed, that's because a combination of lazyness and bad copy/paste behaviour :-)
I'll try to be more consistent.